test/fixtures/recursive-a.cjs

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+global.counter ??= 0;
+global.counter++;
```

Comment:
You can avoid using a global if it's better :)
```suggestion
exports.counter ??= 0;
exports.counter++;
```

Reply:
I don't think it's better, I'm trying to catch a bug where the same module is loaded more than once. `exports` could be different on the second execution, `global` will be the same no matter what.